src/atoms/tableScope/rowActions.ts

```diff
@@ -423,6 +425,14 @@ export const updateFieldAtom = atom(
         deleteFields: deleteField ? [fieldName] : [],
       });
 
+      // TODO(han): Formula field persistence
+      // const config = find(tableColumnsOrdered, (c) => {
+      //  const [, key] = fieldName.split("_rowy_formulaValue_");
+      //  return c.key === key;
+      // });
+      // if(!config.persist) return;
+      if (fieldName.startsWith("_rowy_formulaValue")) return;
+
```

Comment:
Hey @mnmt7, I've just added it, it's required to prevent unnecessary db updates.